src/app/app.module.ts

```diff
@@ -100,7 +108,8 @@ import { PandaGodrollsService } from './service/panda-godrolls.service';
     PlayerStateService,
     PreferredStatService,
     StorageService,
-    WeekService
+    WeekService,
+    LoggedInGuard
```

Comment:
Ended up not using this guard, but it's still a handy guard to have. It checks to see if there's a currently authenticated user.